Hack the linker flags for shared libs for speed and memory usage

---
 tools/llvm-shlib/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/llvm-shlib/Makefile b/tools/llvm-shlib/Makefile
index 6d6c6e9..4038df4 100644
--- a/tools/llvm-shlib/Makefile
+++ b/tools/llvm-shlib/Makefile
@@ -76,7 +76,7 @@ endif
 
 ifeq ($(HOST_OS), $(filter $(HOST_OS), Linux GNU))
     # Don't allow unresolved symbols.
-    LLVMLibsOptions += -Wl,--no-undefined
+    LLVMLibsOptions += -Wl,--no-undefined -Wl,-Bsymbolic
 endif
 
 ifeq ($(HOST_OS),SunOS)
-- 
1.8.3.1

